docs/extensions/cross-chain-data/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
+    "label": "Event-Driven Oracles",
```

Comment:
Can we rename the folder this enitre section is in as "event-driven-oracles"? Makes it easier to navigate in vscode if the folder matches the section name.

Right now, it is "cross-chain data".